shared/idmap/empty.go

```diff
@@ -0,0 +1,3 @@
+package idmap
+
+// Empty file so that builds on !linux don't complain about empty package
```

Comment:
We'll probably want to rename that to doc.go and put some useful content inside it, but fine for now.